### DISCUSSION TITLE
GRR V4.3.3 - Premier login "ADMINISTRATEUR"

### DISCUSSION THREAD

**Discussion by Redhook** [JeromeDevome/GRR#367] [Q&A] [answered]
Bonjour,
Tout d'abord, merci pour ce merveilleux logiciel qu'est GRR.

Je viens de réaliser un installation toute fraiche en v4.3.3
L'installation c'est bien passée de bout en bout.
Tout les pré-requis était au vert.

Lors du premier login en "ADMINISTRATEUR", il ne ce passe rien.
Le mot de passe est correct (pas de message indiquant le contraire).

L'url change de "/login.php" à "/login.php?url=%2Fday.php%3Farea%3D1" , mais l'affichage reste sur la page de login.

J'ai loupé quelque chose ?
Cordialement.

**Comment by JeromeDevome** [comment, marked as answer]
Bonjour,
Vous êtes sur quel fuseau horaire et votre serveur est à qu'elle heure ?
Si vous ou votre serveur n'êtes pas sur l'heure de paris, il faut changer la timezone dans le fichier config.inc.php

**Reply by Redhook** [reply]
Bonjour,
J'utilise GRR depuis des années en métropole, et je n'ai jamais été déçu.
En effet, je suis depuis peu affecté sur l'ile de la Réunion.

J'ai fait la modification:
vim /var/www/html/include/config.inc.php
date_default_timezone_set('Indian/Reunion');

Tout est parfaitement fonctionnel.
Je n'aurai jamais trouvé tout seul, car je n'ai jamais eu a changer de fuseau horaire.

Je ne peux que vous remercier encore, pour le réponse et l'ensemble de votre travail.

Cordialement.
BC
